File: class_tool/src/main/java/com/tempsfteam/class_tool/dto/ProfessionDTO.java
package com.tempsfteam.class_tool.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;


/**
 * @author hypocodeemia
 */
@Data
@AllArgsConstructor
@NoArgsConstructor
public class ProfessionDTO {
    private Integer professionId;

    private Integer courseTypeId;

    private String name;
}

File: class_tool/src/main/resources/static/src/models/dto/professionDTO.js
export default {
    name: 'professionDTO',
    title: 'ProfessionDTO',
    fields: [
        { name: 'professionId', label: '专业ID', type: 'number' },
        { name: 'name', label: '专业名称', type: 'string', required: true },
        { name: 'courseTypeId', label: '课程类型ID', type: 'number' }
    ]
};